cpp/tests/copying/gather_struct_tests.cu

```diff
@@ -0,0 +1,458 @@
+/*
+ * Copyright (c) 2020, NVIDIA CORPORATION.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+#include <cudf/copying.hpp>
+#include <cudf/lists/lists_column_view.hpp>
+#include <cudf/table/table.hpp>
+
+#include <algorithm>
+#include <functional>
+#include <initializer_list>
+#include <iterator>
+#include <memory>
+#include <tests/utilities/base_fixture.hpp>
+#include <tests/utilities/column_utilities.hpp>
+#include <tests/utilities/column_wrapper.hpp>
+#include <tests/utilities/cudf_gtest.hpp>
+#include <tests/utilities/type_lists.hpp>
+#include <tuple>
+#include "cudf/column/column_factories.hpp"
+#include "cudf/detail/utilities/device_operators.cuh"
+#include "cudf/null_mask.hpp"
+#include "cudf/structs/structs_column_view.hpp"
+#include "cudf/table/table_view.hpp"
+#include "cudf/types.hpp"
+#include "cudf/utilities/error.hpp"
+#include "gtest/gtest.h"
+#include "rmm/device_buffer.hpp"
+#include "thrust/host_vector.h"
+#include "thrust/iterator/counting_iterator.h"
+#include "thrust/scan.h"
+#include "thrust/sequence.h"
```

Comment:
I think we always use `<>`?

Reply:
Thanks for pointing this out. I've fixed `structs_column_tests.cu` as well.